### DISCUSSION TITLE
Feature Request: Preview data in the Action modal

### DISCUSSION THREAD

**Discussion by bkuhl** [laravel/nova-issues#5978] [Ideas & Feature Requests]
I would love the ability when adding a field to an action modal to keep the field in "view only" mode, not rending an editable form field, but have it render how it does when viewing a resource.  This would be useful trying to display data, whether it be from the initial action load, or in the below example, a dependent field:

<img width="587" alt="Screenshot 2023-10-20 at 8 41 42 AM" src="https://github.com/laravel/nova-issues/assets/524933/31d37903-5e3f-400d-8bd4-fd506aafe372">

Ideally in this screenshot, instead of a textarea I could show an HTML-formatted preview.

**Comment by davidhemphill** [comment]
This already seems possible (mostly) when using dependent fields. Having each field support its detail mode when editing to support the feature you're describing would be likely prohibitively difficult for a niche feature, ya know? It probably makes the most sense to build a custom field that is solely used for previewing the type of data you need.

**Comment by bkuhl** [comment]
Yeah, I wasn't sure if it would be complex or simple depending on how the front end components are structured.

I tend to avoid custom fields because I want the upgrade path for Nova to be as simple as possible.  But my current approach of just using text area fields that are readonly has been sufficient so far.  If that changes I'll pursue custom fields.
